wagtail/wagtailusers/forms.py

```diff
@@ -17,6 +19,13 @@
     UserPagePermissionsProxy)
 from wagtail.wagtailusers.models import UserProfile
 
+
+DJANGO_GTE_1_9 = django_version >= (1, 9)
```

Comment:
The usually write these version checks like ``django.VERSION >= (1, 9)``. If we follow this convention, it will be easier to search for when we remove old Django versions.